vcd/datasource_vcd_nsxt_alb_pool.go

```diff
@@ -0,0 +1,253 @@
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "Maximum time in minutes to gracefully disable pool member (default 1)",
+			},
+			"member": {
+				Type:        schema.TypeSet,
+				Computed:    true,
+				Description: "ALB Pool Members",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": &schema.Schema{
+							Type:        schema.TypeBool,
+							Computed:    true,
+							Description: "Shows is the member is enabled or not",
+						},
+						"ip_address": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Name of pool member",
+						},
+						"port": {
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Description: "Service port",
+						},
+						"ratio": {
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Description: "Load ratio",
+						},
+						"marked_down_by": {
+							Type:        schema.TypeSet,
+							Computed:    true,
+							Description: "Marked down by",
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"health_status": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Health status",
+						},
+						"detailed_health_message": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Detailed health message",
+						},
+					},
+				},
+			},
+			"health_monitor": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": &schema.Schema{
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Type of health monitor",
+						},
+						"name": &schema.Schema{
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"system_defined": &schema.Schema{
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"persistence_profile": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "System generated name of persistence profile",
+						},
+						"type": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Type of persistence strategy",
+						},
+						"value": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Value of attribute based on persistence type",
+						},
+					},
+				},
+			},
+			"ca_certificate_ids": {
+				Type:        schema.TypeSet,
+				Computed:    true,
+				Description: "A set of root certificate IDs to use when validating certificates presented by pool members",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"cn_check_enabled": &schema.Schema{
```

Comment:
would be check with team if `cn_check_enabled` or `common_name_check_enabled`

Reply:
cc @lvirbalas, @dataclouder

Reply:
This one is tricky. We need to look at all certificate related fields:
- `ca_certificate_ids`
- `cn_check_enabled`
- `domain_names`

In this light, "CN" is as common abbreviation as "CA" is, but I agree that `common_name_check_enabled` is clearer. It kind of becomes consistent with `domain_names` field then:
- `ca_certificate_ids`
- `common_name_check_enabled`
- `domain_names`

Alternatively, maybe there's a way to group all of them with some short prefix?

CC: need @dataclouder input too.

Reply:
This morning the `cn_check_enabled` look fine to me :)

Reply:
I prefer `common_name_check_enabled`